layouts/partials/css.html

```diff
@@ -1,37 +1,40 @@
+<!-- All former base stylesheet links commented out as we are using Docsy for styles. -->
+
 {{ $inServerMode := site.IsServer }}
-{{ $sass         := "sass/style.sass" }}
-{{ $cssOutput    := "css/style.css" }}
-{{ $devOpts      := (dict "targetPath" $cssOutput "enableSourceMap" true) }}
-{{ $prodOpts     := (dict "targetPath" $cssOutput "outputStyle" "compressed") }}
-{{ $cssOpts      := cond $inServerMode $devOpts $prodOpts }}
-{{ $css          := resources.Get $sass | resources.ExecuteAsTemplate $sass . | toCSS $cssOpts }}
-{{ if $inServerMode }}
-<link rel="stylesheet" href="{{ $css.RelPermalink }}">
-{{ else }}
-{{ $prodCss := $css | fingerprint }}
-<link rel="stylesheet" href="{{ $prodCss.RelPermalink }}" integrity="{{ $prodCss.Data.Integrity }}">
-{{ end }}
 
 <link rel="stylesheet" href="/css/base_fonts.css">
+
 {{- if .Params.case_study_styles }}
-<link rel="stylesheet" href="/css/case-study-styles.css">
+  {{ $cssOutput := "css/case-studies.css" }}
+  {{ $caseStudiesCSS := resources.Get "scss/_case-studies.scss" | resources.ToCSS }}
```

Comment:
Can you explain this change. Thanks!

Reply:
Sure! :) The existing `style.sass` includes:
```
{{ if .Params.case_study_styles }}
@import "case-studies"
{{ end }}
```
Since Docsy isn't set up to process `_variables_project.scss` as a template, this was moved here. Additionally, `case-study-styles.css` did not appear to be present in the repo anymore.